changelog.d/16332.feature

```diff
@@ -0,0 +1,2 @@
+Added support for pydantic v2 in addition to pydantic v1.
+Contributed by Maxwell G (@gotmax23).
```

Comment:
Our changelog generator doesn't like multiple lines:

```suggestion
Added support for pydantic v2 in addition to pydantic v1. Contributed by Maxwell G (@gotmax23).
```

Reply:
FTR, it worked just fine when I ran `towncrier build` locally. Both markdown and RST support semantic linebreaks.